client/src/app/components/client-gridview/client-gridview.component.html

```diff
@@ -0,0 +1,13 @@
+<table>
+    <tr>
+        <th>First Name</th>
+        <th>Is Active</th>
+        <th>Last Name</th>
+        <th>Role</th>
+    </tr>
+    <tr *ngFor="let item of users">
+        <td *ngFor="let item1 of item | keyvalue">
+            {{item1.value}}
+        </td>
+    </tr>
+</table>
```

Comment:
If you are working on the frontend, you should use angular material. In this particular case, there is a component for tables ( https://material.angular.io/components/table/overview ) please implement it. I'm asking that because @makchamp already did the basic functionality to see the user in the frontend so redoing that without making it pretty is not really relevant because it will need to be refactored later (extra work).

Reply:
I'm not sure what you're referring too, I first coded the obtaining of the client information in this commit: 0ce7e2150cf1628464a80412fe080f2efbb3231a from a previous PR and then followed up in the current PR by creating a basic table in commit 6594d4b71879ebbb244c23ebc4854382134914f9 . Please look at the commits before commenting. As of commit ef691e7f16ed2ed697ba24007f9fd40a58b457eb angular material has been used

Reply:
I looked at the commits before commenting, you should probably look at the comments i add before doing changes, because at first you just had a simple table that was not following the convention.

Reply:
> 
> 
> I looked at the commits before commenting, you should probably look at the comments i add before doing changes, because at first you just had a simple table that was not following the convention.

That was literally the point of this PR and **that I made**, not @makchamp , making it alone relevant and making the refactoring of it unnecessary as per your first comment, however as per your request this has been resolved